app/assets/stylesheets/nucore-overrides.scss

```diff
@@ -9,3 +9,21 @@ Example of this to include here:
   $headerBackgroundColor: #571963;
   $headerColor: white;
 */
+
+// Shared
```

Comment:
I feel like these variables could probably move to `nucore-variables.scss`. All of these seem like more generic variables for colors, so there's no harm in keeping them around. Usually, we like to reserve this file for changes in the fork and leave this empty in `nucore-open`.

One thing to think about while use/manage mode is if there's a way to define and use variables like `$manageColor`, `$manageLightColor` (or ideally better names). Then you would have something like `$manageColor: $orange`, but other schools could use a different color scheme but still be able to use the use/manage feature.